safe/impact_functions/generic/continuous_hazard_population/parameter_definitions.py

```diff
@@ -0,0 +1,15 @@
+# coding=utf-8
+from safe_extras.parameters.input_list_parameter import InputListParameter
+
+__author__ = 'lucernae'
+__date__ = '13/04/15'
+
+
+def categorical_thresholds():
```

Comment:
Again the semantics here seem wrong - we are working with continuous data so using the overloaded term 'Categorical' here is not good.